suite-native/app/eas.json

```diff
@@ -0,0 +1,57 @@
+{
+    "cli": {
+        "version": ">= 7.1.1",
+        "promptToConfigurePushNotifications": false,
+        "appVersionSource": "remote"
+    },
+    "build": {
+        "develop": {
+            "autoIncrement": true,
+            "env": {
+                "ENVIRONMENT": "develop"
+            },
+            "ios": {
+                "distribution": "store"
+            },
+            "android": {
+                "distribution": "internal"
+            }
+        },
+        "production": {
+            "autoIncrement": true,
+            "env": {
+                "ENVIRONMENT": "production"
+            },
+            "android": {
+                "credentialsSource": "remote"
+            },
+            "ios": {
+                "credentialsSource": "remote"
+            }
+        }
+    },
+    "submit": {
+        "develop": {
+            "android": {
+                "track": "internal",
+                "changesNotSentForReview": true
+            },
+            "ios": {
+                "bundleIdentifier": "io.trezor.suite",
```

Comment:
not sure, but should not be bundle identifier different for `develop`? Something likne `io.trezor.suite.develop`?

Reply:
Fixed